ui/components/app/open-sea-whats-new-popover/open-sea-whats-new-popover.js

```diff
@@ -0,0 +1,124 @@
+  const history = useHistory();
+
+  const hasThePopoverBeenShown = useSelector(
+    getHasTheOpenSeaTransactionSecurityProviderPopoverBeenShown,
+  );
+
+  return (
+    process.env.TRANSACTION_SECURITY_PROVIDER &&
+    !hasThePopoverBeenShown && (
+      <Popover
+        title={
+          <Text
+            variant={TextVariant.headingSm}
+            color={TextColor.textAlternative}
+            fontWeight={FONT_WEIGHT.BOLD}
+          >
+            {t('staySafeWithOpenSea')}
+          </Text>
+        }
+        footer={
+          <>
+            <Button
+              type="primary"
+              onClick={() => {
+                dispatch(setTransactionSecurityCheckEnabled(true));
+                dispatch(
+                  setOpenSeaTransactionSecurityProviderPopoverHasBeenShown(),
+                );
+                history.push(
+                  `${EXPERIMENTAL_ROUTE}#transaction-security-check`,
+                );
+              }}
+              className="open-sea-whats-new-popover__enable-security-provider-button"
+            >
+              {t('enableOpenSeaSecurityProvider')}
+            </Button>
+            <Box marginTop={2}>
+              <Text variant={TextVariant.bodySm}>
+                <Button
+                  type="link"
+                  onClick={() =>
+                    dispatch(
+                      setOpenSeaTransactionSecurityProviderPopoverHasBeenShown(),
+                    )
+                  }
+                >
+                  {t('notNow')}
+                </Button>
+              </Text>
+            </Box>
+          </>
+        }
+        footerClassName="smart-transactions-popover__footer"
+        className="smart-transactions-popover"
+        onClose={() =>
+          dispatch(setOpenSeaTransactionSecurityProviderPopoverHasBeenShown())
+        }
+      >
+        <Box
+          paddingRight={6}
+          paddingLeft={6}
+          paddingTop={0}
+          paddingBottom={0}
+          display={DISPLAY.FLEX}
+          className="smart-transactions-popover__content"
+        >
+          <Box
+            marginTop={1}
+            marginBottom={1}
+            display={DISPLAY.FLEX}
+            flexDirection={FLEX_DIRECTION.COLUMN}
+          >
+            <img
+              src="./images/open-sea-security-provider.svg"
+              alt={t('openSeaAltText')}
+            />
+          </Box>
+          <Text variant={TextVariant.bodySm}>
+            {t('getWarningsFromOpenSea')}
+          </Text>
+          <Text
+            variant={TextVariant.bodySm}
+            marginTop={4}
+            fontWeight={FONT_WEIGHT.BOLD}
+          >
+            {t('openSeaDescription')}
+          </Text>
+          <Text
+            variant={TextVariant.bodySm}
+            color={TextColor.textAlternative}
+            marginTop={4}
+          >
+            {t('alwaysBeSureTo')}
+          </Text>
+        </Box>
+      </Popover>
+    )
+  );
+}
```

Comment:
Will be nice to have some test coverage for this component.

Reply:
I agree. I created this PR some time ago when it wasn't mandatory... 
I'll cover it with tests after I finish all the tickets regarding the security provider feature. For now, I think we can push forward this PR and not block it, if you agree.